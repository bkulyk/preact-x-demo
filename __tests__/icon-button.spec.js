import React from 'react';
import IconButton from '@bitchin/icon-button';
import { render } from './enzyme';
import { it, expect } from './jest';

it('should render correctly', () => {
  expect(render(<IconButton icon='favorite'>Favorite</IconButton>))
    .toMatchSnapshot();
});