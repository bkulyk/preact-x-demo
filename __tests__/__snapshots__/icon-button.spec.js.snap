// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<button
  aria-label="favorite"
  class="mdc-icon-button material-icons mdc-top-app-bar__action-item"
  type="button"
>
  favorite
</button>
`;
